simvue/executor.py

```diff
@@ -203,10 +208,20 @@ def callback_function(status_code: int, std_out: str, std_err: str) -> None:
             )
 
         if script:
-            self._runner.save_file(file_path=script, category="code")
+            if cwd:
```

Comment:
Can simplify this to:

```python
self._runner.save_file(
    file_path=os.path.join(cwd or os.getcwd(), script), category="code"
)
```

Reply:
My issue with this would be what if they've provided the absolute path to the script? Will it not then duplicate the start of the path and fail to find the file?

Reply:
Well the issue then is that someone could provide both `cwd` and the path to their script as an absolute path. 
I don't really agree with script being a string this creates ambiguity:

* `cwd` should be directory to run process in.
* `script` should be a path to the script to execute.

The two are not related as you might want to run a script from another directory in the current directory. I think the solution here is unstable